Makefile

```diff
@@ -29,9 +29,13 @@ destroy: down ## Bring down the Kafka2Hbase Docker container and services then d
 
 .PHONY: integration
 integration: ## Run the integration tests in a Docker container
-	docker-compose run --rm integration-test ./gradlew --rerun-tasks integration
```

Comment:
Do we want to add this back in now?

Reply:
I have only  removed --rm as we dont want to remove the container as soon as exited to be able to view logs